test/controllers/images_controller_test.rb

```diff
@@ -61,5 +61,14 @@ def test_should_show_image
     get image_path(id: testimage.id)
     assert_response :ok
     assert_select "img[src='http://abc.png']", count: 1
+    assert_select '.tags-list', count: 0 # no tags block
+  end
+
+  def test_should_show_image_with_tag
```

Comment:
This is a good start cause it tests the changes in the `show` template. However, we should also test the `create` action that is affected by this PR as well.